src/main/java/de/zalando/zmon/dataservice/config/LightStepConfig.java

```diff
@@ -0,0 +1,33 @@
+package de.zalando.zmon.dataservice.config;
+
+import io.opentracing.Tracer;
+import com.lightstep.tracer.jre.JRETracer;
+import com.lightstep.tracer.shared.Options;
+
+import java.net.MalformedURLException;
+
+public class LightStepConfig {
```

Comment:
I see an interface or abstract class here. Both JaegerConfig and LightStepConfig hold a Tracer and export the getTracer() method...

Reply:
So something like this should do:
<package de.zalando.zmon.dataservice.config;
import io.opentracing.Tracer;
public interface OpenTracerConfig {
    public Tracer generateTracer();
}
>
and 
< public Tracer generateTracer(){
        Tracer tracer = new Configuration(serviceName,
                new Configuration.SamplerConfiguration(ProbabilisticSampler.TYPE, 1),
                new Configuration.ReporterConfiguration(logSpans, jaegerHost, jaegerPort, flushIntervalMs, maxQueueSize)).getTracer();
        return tracer;
    }>

Reply:
I dont even need to hold the tracer in the config as it can be extracted from opentracer's Globaltracer